<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sponsoring</class>
 <widget class="QDialog" name="sponsoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sponsoring</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(4, 15, 19);

color: #aa54ff; 
}
#filter,#mediaplayer,#notif,#pdf{
background-color: transparent;
}


#centralwidget{
background-color: #040f13;
}
#sidemenu{
background-color: #071e26; 
border-radius: 20px; 
}
#ajouter,#supprimer,#modifier
{
padding: 20px; 
background-color: #7f3fbf; 
border-radius: 5px;
color : #fff;
}
QPushButton,QLineEdit{
padding: 20px; 
background-color: #040f13; 
border-radius: 5px; }
#header{
background-color: #071e26; 
border-radius: 10px;
}
#rightsidemenu
{
background-color: #071e26; 
border-radius: 20px; 
}

#mainbody{
background-color: #071e26; 
border-radius: 10px;
}</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>120</y>
     <width>1111</width>
     <height>601</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="sidemenu">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>181</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>  publication</string>
     </property>
    </widget>
    <widget class="QLabel" name="research_pic">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../2a3-2a3-smart-rdi-center-sponsor - Copie/icons/research.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>sponsor</string>
     </property>
    </widget>
    <widget class="QLabel" name="sponsor_pic">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../2a3-2a3-smart-rdi-center-sponsor - Copie/icons/handshake.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>materiel</string>
     </property>
    </widget>
    <widget class="QLabel" name="materiel_pic">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../2a3-2a3-smart-rdi-center-sponsor - Copie/icons/flask.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>  encadrant</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/teacher.png</normaloff>icons/teacher.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>etudiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="student_pic">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../2a3-2a3-smart-rdi-center-sponsor - Copie/icons/student.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="encadrant_pic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../teacher.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="mainbody">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>581</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>50</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>-10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Liste des sponsors</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>39</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf_pic">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>280</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri_pic">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>521</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rech_pic">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="tri_pic_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Tri Duree</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Tri Montant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri_pic_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Tri ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push0">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>390</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="rightsidemenu">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>40</y>
      <width>201</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter un sponsor</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>116</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Duree</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>116</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>id a supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>116</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Montant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Contact</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>116</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ID_sponsor</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="header">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="hamburger_pic">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>10</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>50</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/hamburger.png</normaloff>icons/hamburger.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>20</height>
     </size>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>50</horstretch>
      <verstretch>50</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>1</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>InteLab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>sta</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pdf/icons/statistic.png</normaloff>pdf/icons/statistic.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="calendrier_pic">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>calendrier</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pdf/icons/calendar-date.png</normaloff>pdf/icons/calendar-date.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="camera_pic">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>camera</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/camera.png</normaloff>icons/camera.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
